lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
+    const { host, port } = lndCLient.config;
+
+    lndCLient.lightning = new LightningClient(`${host}:${port}`, lndCLient.credentials);
+
+    lndCLient.getInfo().then((getInfoResponse) => {
+      if (getInfoResponse && getInfoResponse.numActiveChannels >= 1) {
+        lndCLient.setStatus(ClientStatus.CONNECTION_VERIFIED);
+        if (subscribeInvoices) {
+          lndCLient.subscribeInvoices();
+        }
+      }
+    }).catch((err: any) => {
+      lndCLient.logger.error(`could not fetch info from lnd host, error: ${err}, retrying in 5000 ms`);
+      lndCLient.setStatus(ClientStatus.DISABLED);
+      setTimeout(lndCLient.connect.bind(null, lndCLient), 5000);
```

Comment:
I'm not 100% sure, but I don't believe we need the `bind` statement here because we're using arrow functions.

Reply:
already removed, because of arrow function as you mentioned we don't need lndClient anymore.